src/index.md

```diff
@@ -79,18 +79,18 @@ and [spaces](features.md#higher-order-finite-element-spaces).
 * ... and [many more](features.md).
 
 MFEM is used in many projects, including [BLAST](http://www.llnl.gov/casc/blast), [Cardioid](https://github.com/llnl/cardioid), [VisIt](http://visit.llnl.gov), [RF-SciDAC](https://www.rfscidac4.org/), [FASTMath](https://fastmath-scidac.llnl.gov/index.html), [xSDK](https://xsdk.info/), and [CEED](http://ceed.exascaleproject.org) in the [Exascale Computing Project](https://exascaleproject.org).
-See also our [Gallery](gallery.md), [Publications](publications.md) and [News](news.md) pages.
+See also our [Gallery](gallery.md), [Publications](publications.md), [Videos](videos.md), and [News](news.md) pages.
 
 </div><div class="col-md-6 news-table" markdown="1">
 
 ## News
 
 Date         | Message
 ------------ | -----------------------------------------------------------------
+Nov 30, 2021 | New [page](videos.md) with recorded talks + videos.
```

Comment:
At some point we may want to add the videos and workshop pages to the drop down interface.

Reply:
I was thinking about that too... where will you put them?

Reply:
Maybe on the bottom after the virtual backgrounds?

Reply:
Maybe documentation along with publications?

Reply:
I'm OK with this. Do you want to make the change?

Reply:
Yup, I'll push it in now.